### ISSUE TITLE
Not able to reproduce results with the pre-trained DeepAugment + AugMix model

### ISSUE THREAD

**Issue by LeonGuertler** [hendrycks/imagenet-r#7]
Hey there, 

first of all, thank you for publishing the code to the paper! 

I downloaded the "DeepAugment + AugMix" model you provide and evaluated it on the ImageNet-c testset using the mCE metric (I took the evaluation code directly from https://github.com/bethgelab/game-of-noise) and get a mCE score of 69 (which differs from the ~53 reported). I am not sure if something is wrong on my side, and just want to double-check how you calculated the mCE.
